### ISSUE TITLE
Install Issue: node-sass fails to install.

### ISSUE THREAD

**Issue by k-funk** [yeoman/generator-backbone#366]
npm ERR! Darwin 14.5.0
    npm ERR! argv "/usr/local/Cellar/node/4.1.0/bin/node" "/usr/local/bin/npm" "install"
    npm ERR! node v4.1.0
    npm ERR! npm  v2.14.4
    npm ERR! code ELIFECYCLE

    npm ERR! node-sass@1.2.3 postinstall: `node scripts/build.js`
    npm ERR! Exit status 1
    npm ERR!
    npm ERR! Failed at the node-sass@1.2.3 postinstall script 'node scripts/build.js'.
    npm ERR! This is most likely a problem with the node-sass package,
    npm ERR! not with npm itself.
    npm ERR! Tell the author that this fails on your system:
    npm ERR!     node scripts/build.js
    npm ERR! You can get their info via:
    npm ERR!     npm owner ls node-sass
    npm ERR! There is likely additional logging output above.

Happens when installing with bootstrap for sass. Also, I thought node-sass was on version 3.x. Why is 1.2.3 trying to install?

**Comment by arthurvr** [comment]
What version of the generator did you use? Can you try using master? Thanks for the report.

**Comment by k-funk** [comment]
`npm install -g generator-backbone` fails at `yo backbone`, as mentioned above:
```
generator-backbone@0.3.5 /usr/local/lib/node_modules/generator-backbone
    └── yeoman-generator@0.17.7 (dargs@2.1.0, diff@1.4.0, isbinaryfile@2.0.4, class-extend@0.1.1, async@0.9.2, underscore.string@2.4.0, mime@1.3.4, text-table@0.2.0, lodash@2.4.2, debug@1.0.4, grouped-queue@0.3.0, nopt@3.0.4, iconv-lite@0.4.11, cross-spawn@0.2.9, chalk@0.5.1, shelljs@0.3.0, run-async@0.1.0, mkdirp@0.5.1, glob@4.5.3, file-utils@0.2.2, rimraf@2.4.3, findup-sync@0.1.3, github-username@1.1.1, download@1.0.7, gruntfile-editor@0.2.0, cheerio@0.17.0, request@2.64.0, inquirer@0.7.3)
```

When installing from master with `npm install -g git+https://github.com/yeoman/generator-backbone.git` it  works at `yo backbone`:
```
generator-backbone@0.3.5 /usr/local/lib/node_modules/generator-backbone
├── mkdirp@0.5.1 (minimist@0.0.8)
├── param-case@1.1.1 (sentence-case@1.1.2)
├── pascal-case@1.1.1 (upper-case-first@1.1.1, camel-case@1.1.2)
├── ejs@2.3.4
├── html-wiring@1.2.0 (detect-newline@1.0.3, cheerio@0.19.0)
└── yeoman-generator@0.20.3 (read-chunk@1.0.1, yeoman-assert@2.1.0, detect-conflict@1.0.0, path-exists@1.0.0, yeoman-welcome@1.0.1, async@1.4.2, path-is-absolute@1.0.0, rimraf@2.4.3, mime@1.3.4, text-table@0.2.0, dargs@4.0.1, user-home@2.0.0, xdg-basedir@2.0.0, istextorbinary@1.0.2, debug@2.2.0, nopt@3.0.4, diff@2.1.2, chalk@1.1.1, run-async@0.1.0, cli-table@0.3.1, class-extend@0.1.1, shelljs@0.5.3, through2@2.0.0, glob@5.0.14, pretty-bytes@2.0.1, yeoman-environment@1.2.7, findup-sync@0.2.1, underscore.string@3.2.2, dateformat@1.0.11, mem-fs-editor@2.0.4, github-username@2.0.0, sinon@1.17.0, cross-spawn@2.0.0, gruntfile-editor@1.0.0, download@4.2.1, lodash@3.10.1, inquirer@0.8.5)
```

...time for a version bump?

**Comment by arthurvr** [comment]
Yep, https://github.com/yeoman/generator-backbone/issues/357. If someone else can confirm everything else stayed working on master I'm happy to do a release, but I don't use this generator myself.

**Comment by abwaters** [comment]
This issue is still happening for me.

**Comment by ruyadorno** [comment]
@abwaters are you using latest release `v0.3.5` or did you tried installing from the `master` branch?

**Comment by abwaters** [comment]
I've tried installing from the master branch but i'm not convinced that i've done my homework.  I did a clean install of yeoman, and the generators and i install node-sass cleaning as a global module but the backbone generation fails with the error above regardless of whether I use the master or the release version of the backbone generator.  

It seems to be pulling in the 1.2.3 dependency even though I have node-sass@3.4.2 cleanly installed as a global.

I'm still researching though...
